.github/workflows/sycl-linux-build.yml

```diff
@@ -186,6 +186,15 @@ jobs:
         # TODO consider moving this to Dockerfile.
         export LD_LIBRARY_PATH=/usr/local/cuda/compat/:/usr/local/cuda/lib64:$LD_LIBRARY_PATH
         cmake --build $GITHUB_WORKSPACE/build --target check-sycl
+    - name: check-sycl-unittests
+      if: always() && !cancelled() && contains(inputs.changes, 'sycl')
+      run: |
+        # TODO consider moving this to Dockerfile.
+        export LD_LIBRARY_PATH=/usr/local/cuda/compat/:/usr/local/cuda/lib64:$LD_LIBRARY_PATH
+
+        export LD_LIBRARY_PATH=$GITHUB_WORKSPACE/build/lib:$LD_LIBRARY_PATH
+        export PATH=$GITHUB_WORKSPACE/build/bin:$PATH
```

Comment:
This needs a comment explaining the issue and why we deal with it this way and what other alternatives were rejected.